mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/impl/SimpleMailboxMessageTest.java

```diff
@@ -59,6 +59,7 @@ public void testSize() {
 
     @Test
     public void testInputStreamSize() throws IOException {
+        @SuppressWarnings("resource")
```

Comment:
because an bytearrayoutputstream don't really leak, but, hey, it's not really a part of my work, I'll rebase on antoine's work